src/resource-graph/azext_resourcegraph/azext_metadata.json

```diff
@@ -1,4 +1,4 @@
 {
-    "azext.minCliCoreVersion": "2.21.0",
-    "azext.isPreview": false
```

Comment:
Why you change it back to preview state again?

Reply:
Because the shared queries functionality is only in preview, we didn't GAed on it. Let me know if I should revert it back to isPreview false anyways.

Reply:
You’ve already marked shares queries commands as experimental. So you don’t need to change here.

Reply:
Reverted back to false